src/Hint/Monad.hs

```diff
@@ -40,9 +40,9 @@ folder f a xs = foldM f a xs >>= \_ -> return () -- foldM_ f a xs
 yes = mapM async ds >>= mapM wait >> return () -- mapM async ds >>= mapM_ wait
 main = "wait" ~> do f a $ sleep 10
 main = print do 17 + 25  @NoRefactor: needs -XBlockArguments which isn't available before GHC 8.6
-main = print do 17 -- 17 @NoRefactor
-main = f $ do g a $ sleep 10 -- g a $ sleep 10
-main = do f a $ sleep 10 -- f a $ sleep 10
+main = print do 17 -- @NoRefactor
```

Comment:
Why is this one NoRefactor?

Reply:
apply-refact (0.6.0.0) crashes if the BlockArgument pragma is absent.

Reply:
Ah, cool - I note the line above even explains it. I copied the explanation on to that line too. Thanks.